### ISSUE TITLE
Buffer app not working 

### ISSUE THREAD

**Issue by Quorum75** [StevenBlack/hosts#977]
Hello. 
Since the last update the Buffer iOS app (Buffer: Social Media Manager by Buffer, Inc.) is not working properly anymore. 
The fetching link function does work. 
Best. 
James

**Comment by dnmTX** [comment]
@Quorum75  see if you can comment or remove  `0.0.0.0 bufferapp.com`  test it and post results please.

**Comment by Quorum75** [comment]
> @Quorum75 see if you can comment or remove `0.0.0.0 bufferapp.com` test it and post results please.

Thank you! I don't know how I missed it even with a search. 

I don't understand why the app is in the list as there is nothing wrong with. Also, it is not consistent that other similar apps are whitelisted like Hootsuite...

**Comment by dnmTX** [comment]
@Quorum75  if you don't mind post here all the blocked domains from genuine apps that you think needs whitelisting.Thanks.

**Comment by dnmTX** [comment]
ping @brianclemens  👉  `0.0.0.0 bufferapp.com`  also stand by for @Quorum75  to list what else is needed to be whitelisted. Thank you.

**Comment by Quorum75** [comment]
@dnmTX 
FYI I'm using: "Unified hosts + fakenews + gambling + porn" because I wanted to be able to use social media platforms like LinkedIn. 

Buffer, Hootsuite, Crowdfire apps allow connections to social networks profiles like LinkedIn, Twitter... to easily post news. 
So I guess if you want to block all social networks it is OK, but within the Host file where you have "Social". What do you think?

**Comment by dnmTX** [comment]
@Quorum75  ok,let me rephrase that. The reason i asked you to post here all the blocked domains from genuine apps that you think needs to be whitelisted is because as of late there is a new addition to the hosts from @tiuxo and we trying to weed out all the false positives (it's understandable,with every new lists there will be some bad apples).
So...if you think that there is something else from that specific [LIST](https://raw.githubusercontent.com/tiuxo/hosts/master/ads) please post it here so could be examine and possibly whitelisted. For any other inquiries refer to the curators of the lists posted on the main page here and make your case there.

**Comment by brianclemens** [comment]
Fixed!

**Comment by dnmTX** [comment]
@Quorum75  you can close this if you want,it will be fixed here with the next update.

**Comment by StevenBlack** [comment]
That release is out now.  So closing.

**Comment by ay8s** [comment]
@StevenBlack Hey! I'm one of the iOS Developers at Buffer.

I've been digging into a few issues some of our users have been having with authenticating with Buffer. I'm able to see api.bufferapp.com listed in the hosts file but it looks like it may have been removed in this PR in the past?

Is there another source which pulled in api.bufferapp.com that needs updating? 

Would love to iron out the issues users are seeing with signin. Been seeing it myself with Pihole but only now decided to dig into it.

**Comment by StevenBlack** [comment]
Hi Andrew @ay8s the `api.bufferapp.com` domains comes to us via AdAway.

Ping @jawz101, can you have a look at this one? Thanks.

**Comment by ay8s** [comment]
Thanks @StevenBlack. 👍 

@jawz101 Let me know if I can help at all with this, can also reach me at andy@buffer.com.

**Comment by jawz101** [comment]
god I hate social media marketing shit.  but I'll remove it since I guess they don't like others' ads yet they want to keep theirs

**Comment by StevenBlack** [comment]
So @jawz101 please understand, Andrew @ay8s is here asking nicely, but I trust your judgment one hundred percent.  Please don't feel like you have to, and please don't feel pressured.  Evaluate this on its own merits, use your judgment, and I'm happy.

Always remember who we're serving, here.  It's not Andrew.  It's people who don't want to be tracked, served, sold, or abused, for any reason.

**Comment by ay8s** [comment]
Thanks @StevenBlack. @jawz101 also happy to discuss if you feel like we've done or do anything that would merit Buffer being included in this list and I can see how we can improve.
